PIL/Image.py

```diff
@@ -1236,6 +1307,7 @@ def getpalette(self):
             return None  # no palette
 
     def getpixel(self, xy):
+        # type: (Coord) -> Tuple[Any]
```

Comment:
The issue here is that we're returning a tuple, but whose length varies - though actually also the 'color' type, which I'm not sure how to type offhand?

So, if we alias `Color = [something]`, then ideally the function would return `Union[Color, Tuple[Color, Color], Tuple[Color, Color, Color], ...]`... but where we go all the way to the maximum number of layers in the image! Since I'm not sure if that's possible to quantify, I stuck with `Tuple` for now, though it should probably be `Union[Color, Tuple]` for now, for some value (type) of Color.